internal/pkg/types/message.go

```diff
@@ -157,6 +157,7 @@ func (msg *UnsignedMessage) Cid() (cid.Cid, error) {
 
 // OnChainLen returns the amount of bytes used to represent the message on chain.
 func (msg *UnsignedMessage) OnChainLen() uint32 {
+	return 0
```

Comment:
Chain-validation tests run the vm `appleyMessage` method - it calls this function and removing the panic is required for tests to run.